ZincTests/ZincRepoIndexTests.m

```diff
@@ -22,22 +25,22 @@ - (void) _testDictionaryRoundtrip:(ZincRepoIndex*)index
 {
     NSError* error = nil;
     NSDictionary* dict = [index dictionaryRepresentation];
-    ZincRepoIndex* index2 = [ZincRepoIndex repoIndexFromDictionary:dict error:&error];
+    ZincRepoIndex* index2 = [ZincRepoIndex repoIndexFromDictionary:dict fileURL:self.url error:&error];
```

Comment:
`self.url` appears to be `nil` for these tests. That seems like a mistake. Should we require `fileURL` to not be nil or should we handle the `nil` case in case it could happens in a released build?